python/lsst/ts/observatory/model/observation.py

```diff
@@ -0,0 +1,337 @@
+import math
+import numpy
+
+__all__ = ["Observation"]
+
+class Observation(object):
```

Comment:
Maybe `Observation` should subclass `Target`. This way, we don't need to add methods, properties and variables that are already defined there. 

Make sure the duplicated methods are properly removed here and implement only those that changes or does not exists.